plugwise/smile.py

```diff
@@ -150,13 +173,28 @@ def _device_data_climate(
             device_data["active_preset"] = self._preset(loc_id)
 
         # Schedule
-        avail_schedules, sel_schedule, last_active = self._schedules(loc_id)
+        avail_schedules, sel_schedule, sched_setpoints, last_active = self._schedules(
+            loc_id
+        )
         device_data["available_schedules"] = avail_schedules
         device_data["selected_schedule"] = sel_schedule
         if self._smile_legacy:
             device_data["last_used"] = "".join(map(str, avail_schedules))
         else:
             device_data["last_used"] = last_active
+            if self._anna_cooling_present:
```

Comment:
Lets also reverse this, set sane defaults, overwrite if neccesary

Reply:
There's no returning so not so easy...

Reply:
I've made a small improvement.

Reply:
Yeah I was ... seems github is not capturing the line correctly if you click it for commenting